src/zenml/pipelines/base_pipeline.py

```diff
@@ -191,6 +192,16 @@ def _verify_arguments(self, *steps: BaseStep, **kw_steps: BaseStep) -> None:
                     f"already passed as a positional argument."
                 )
 
+            if isinstance(step, BaseStepMeta):
+                raise PipelineInterfaceError(
+                    f"Wrong argument type (`{step_class}`) for argument "
```

Comment:
Great message! Like the clear and simple message.

I would just maybe a test to ensure the error is called when you want it to be called. Other than that LGTM! :-)!

Reply:
Thanks, great suggestion! I added two unit tests to check that a PipelineInterfaceError is raised when creating a pipeline with missing step brackets.